daemon/src/taker_cfd_actor.rs

```diff
@@ -206,28 +208,60 @@ fn setup_contract(
         send_to_maker(SetupMsg::Msg1(Msg1::from(taker_cfd_txs.clone())));
         let msg1 = receiver.recv().await.unwrap().try_into_msg1().unwrap();
 
-        let _lock_desc = lock_descriptor(maker.identity_pk, taker.identity_pk);
-        // let lock_amount = maker_lock_amount + taker_lock_amount;
+        let lock_desc = lock_descriptor(maker.identity_pk, taker.identity_pk);
 
-        let _commit_desc = commit_descriptor(
+        let lock_amount = maker.lock_amount + taker.lock_amount;
+
+        let commit_desc = commit_descriptor(
             (
                 maker.identity_pk,
                 maker_punish.revocation_pk,
                 maker_punish.publish_pk,
             ),
             (taker.identity_pk, rev_pk, publish_pk),
         );
-        let commit_tx = taker_cfd_txs.commit.0;
 
-        let _commit_amount = Amount::from_sat(commit_tx.output[0].value);
+        let taker_cets = taker_cfd_txs.cets;
+        let commit_tx = taker_cfd_txs.commit.0.clone();
+
+        let commit_amount = Amount::from_sat(commit_tx.output[0].value);
 
-        // TODO: Verify all signatures from the maker here
+        // verify commit tx signature
+        verify_adaptor_signature(
+            &commit_tx,
+            &lock_desc,
+            lock_amount,
+            &msg1.commit,
+            &maker_punish.publish_pk,
+            &taker.identity_pk,
+        )
+        .unwrap();
+
+        // verify cet tx signatures
```

Comment:
:upside_down_face: I think we can remove these comments (particularly this one since it's a bit redundant with the name of the function you call just below it).